.github/CONTRIBUTING.md

```diff
@@ -259,10 +259,14 @@ To help you with this, a number of convenience scripts are available:
 * `composer check-all` will run the `cs` + `test` checks in one go.
 * `composer cs` will check for code style violations.
 * `composer cbf` will run the autofixers for code style violations.
-* `composer test` will run the unit tests.
-* `composer coverage` will run the unit tests with code coverage and show a text summary.
+* `composer test` will run the unit tests when using PHP 8.1+/PHPUnit 10+.
+* `composer testlt10` will run the unit tests when using PHP < 8.1/PHPUnit < 10.
```

Comment:
Similar to what I said above, I suggest using `lte` instead of `lt` and hyphens for these composer commands.

Reply:
Good point, I've updated the PR to action this.